tests/php/Service/ExAppSetupCheckRefreshServiceTest.php

```
@@ -0,0 +1,310 @@
+	public function testDisabledAppIsNotProbed(): void {
+		$this->appAPIService->expects(self::never())->method('requestToExApp');
+		$state = $this->runRefresh(
+			['a'],
+			['a' => $this->makeExApp('a', 0)],
+			[],
+		);
+		self::assertArrayNotHasKey('a', $state);
+	}
+
+	public function testInitializingAppIsNotProbed(): void {
+		$this->appAPIService->expects(self::never())->method('requestToExApp');
```

PHPStan (2.2.2)

[error] Call to an undefined static method OCA\AppAPI\Tests\php\Service\ExAppSetupCheckRefreshServiceTest::never().


(staticMethod.notFound)